mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -171,247 +164,281 @@ interface DexItem<T> {
+                    decimalScale: 8,
+                    testID: `your_${pairSymbol}`
+                  }}
+                />
+              )
+            }
+            <PoolPairInfoLine
+              label={`${type === 'available' ? 'Total pooled' : 'Pooled'} ${pair?.tokenA?.displaySymbol}`}
+              value={{
+                text: tokenATotal.toFixed(decimalScale),
+                decimalScale: decimalScale,
+                testID: `${props.testID}_${pair?.tokenA?.displaySymbol}`
+              }}
+            />
+            <PoolPairInfoLine
+              label={`${type === 'available' ? 'Total pooled' : 'Pooled'} ${pair?.tokenB?.displaySymbol}`}
+              value={{
+                text: tokenBTotal.toFixed(decimalScale),
+                decimalScale: decimalScale,
+                testID: `${props.testID}_${pair?.tokenB?.displaySymbol}`
+              }}
+            />
+            {
+              pair.totalLiquidity.usd !== undefined && (
+                <PoolPairInfoLine
+                  label='Total liquidity (in USD)'
+                  value={{
+                    text: pair.totalLiquidity.usd,
+                    decimalScale: 2,
+                    testID: `totalLiquidity_${pairSymbol}`
+                  }}
+                />)
+            }
+            {
+              pair.apr?.total !== undefined &&
+                <PoolPairInfoLine
+                  label='APR'
+                  value={{
+                  text: new BigNumber(isNaN(pair.apr.total) ? 0 : pair.apr.total).times(100).toFixed(2),
+                  decimalScale: 2,
+                  testID: `apr_${pairSymbol}`,
+                  suffix: '%'
+                }}
+                />
+            }
+          </>
+        )
+      }
+    </View>
+)
 }
 
-function PoolPairInfoLine (props: { symbol: string, reserve: string, row: string, decimalScale: number, label: string }): JSX.Element {
-  const Icon = getNativeIcon(props.symbol)
+interface PoolPairInfoLineProps {
+  label: string
+  value: {
+    decimalScale?: number
+    suffix?: string
+    testID: string
+    text: string
+  }
+}
 
+function PoolPairInfoLine (props: PoolPairInfoLineProps): JSX.Element {
   return (
-    <View style={tailwind('flex-row justify-between mb-1')}>
+    <View style={[tailwind('flex-col justify-between mb-3'), { width: '50%' }]}>
       <ThemedText
         dark={tailwind('text-gray-400')}
         light={tailwind('text-gray-500')}
-        style={tailwind('text-sm font-normal')}
+        style={tailwind('text-xs font-normal')}
       >
-        {translate('screens/DexScreen', `${props.label} {{symbol}}`, { symbol: props.symbol })}
+        {props.label}
       </ThemedText>
 
-      <View style={tailwind('flex-row items-center')}>
-        <NumberFormat
-          decimalScale={props.decimalScale}
-          displayType='text'
-          renderText={value => {
-            return (
+      {props.value.decimalScale === undefined
```

Comment:
correct me if i am wrong, but i see all current usage of `PoolPairInfoLineProps` has a non-undefined `decimalScale`, maybe we can remove the the optional syntax and remove this condition check here